### ISSUE TITLE
Select Field missing particular children in the ARIA roles tag

### ISSUE THREAD

**Issue by davidjacob93** [manchesergit/material-ui#117]
Select field component has missing children role tags

![capture](https://user-images.githubusercontent.com/25061353/27959297-c27b7b68-631e-11e7-818f-1e06d5d799f7.PNG)

Buttons should have accessible names. **Adding an aria-label tag to the button 'may' fix the problem.**
https://dequeuniversity.com/rules/axe/1.1/button-name

![capture2](https://user-images.githubusercontent.com/25061353/27959480-7f5481a8-631f-11e7-8a69-8940c88890f9.PNG)

**Comment by djbuckley** [comment]
this is 2 separate issues
closed and new issues opened as #122 and #123
